manager/Makefile

```diff
@@ -1,72 +1,31 @@
 ROOT_DIR := ..
 include $(ROOT_DIR)/Makefile.env
-#include Makefile.env
 
 .PHONY: all
 all: manager
 
 include $(ROOT_DIR)/hack/make-rules/tools.mk
+include $(ROOT_DIR)/hack/make-rules/docker.mk
+include $(ROOT_DIR)/hack/make-rules/verify.mk
+
 DOCKER_NAME ?= manager
```

Comment:
Don't the previous includes define already DOCKER_NAME?

Reply:
This makes `IMG` be defined correctly. You commented before my last commit so maybe it's clearer now.